File: src/programs/functions/six.js
import { useState,useEffect } from "react"
import FirstJava from "../../topics/FirstJava"
import FirstJavaFour from "./four"
import FunctionsSeven from "./seven"
import Functions from "../../topics/Functions"
const FunctionsSix = ({qn,backPage,qn1,qn2}) =>{
    const q7 = "7.Define a method to find out if a number is prime or not."
    const [file,setFile] = useState("Loading Java FIle...")
    const url = window.location.href
    const back = (page) =>{
        backPage(page)
    }
    useEffect(()=>{
    async function fetchFile() {
        try{
    const res = await fetch(`${url}javaPrograms/functions/Six.java`)

    const cont = await res.text();
    
    setFile(cont)
    }
    catch(err){
    console.log(err);
    }

}
 
fetchFile() 
},[])
    return(<div className="program-body"><div className='topics-logo'>{qn}</div>
        <div className='program-content'>


<div className='program-hint program'><pre>{file}</pre></div>

<div className="program-btns">
<div onClick={() => back(<Functions/>)} className='program-btn'>Go To Map</div>
<div onClick={() => back(<FunctionsSeven qn = {q7} backPage={backPage} />)} className='program-btn'>Next</div>
</div>

</div></div>)
}
export default FunctionsSix

File: public/javaPrograms/functions/Six.java
package javaPrograms.functions;

import java.util.Scanner;

class Six{
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n= s.nextInt();
        int area = area(n);
        int circum = circum(n);
        System.out.println("AREA="+area);
        System.out.println("CIRCUMFERENCE="+circum);
    }
static int area(int r){
    int area = 22/7*r*r;
    return area;
}
static int circum(int r){
    int circum = 2*22/7*r;
    return circum;
}
}